activerecord/test/cases/adapters/mysql2/prepared_statements_test.rb

```diff
@@ -0,0 +1,32 @@
+require "cases/helper"
+require "models/developer"
+require "models/computer"
+
+class PreparedStatementsTest < ActiveRecord::Mysql2TestCase
+  fixtures :developers
+
+  def setup
+    @default_prepared_statements = ActiveRecord::Base.connection.instance_variable_get("@prepared_statements")
+    ActiveRecord::Base.connection.instance_variable_set("@prepared_statements", true)
```

Comment:
Is there any way to toggle prepared statements without remotely setting instance variables? :grimacing: